jwst/resample/gwcs_drizzle.py

```diff
@@ -1,7 +1,6 @@
 import numpy as np
 
-from drizzle import util
-from drizzle import cdrizzle
+from drizzle import cdrizzle, utils
```

Comment:
I think our pre-commit is complaining because `utils` is unused and should be removed from the imports